rules/private/copy_file_private.bzl

```diff
@@ -58,12 +58,12 @@ def copy_bash(ctx, src, dst):
         use_default_shell_env = True,
     )
 
-def _common_impl(ctx, is_executable):
+def _impl(ctx):
```

Comment:
There are several rules and implementations here.
It would be easier to read if this were _copy_file_impl

Reply:
Sure, done